_assets/sass/guide-helper.scss

```diff
@@ -1,5 +1,5 @@
 // styles for guide display only (helpers)
-@import 'global/variables';
+@import 'libraries-main';
```

Comment:
this is a maze of 🙈but it's also fine

Reply:
I know - but it seemed less of a maze than making a whole new css directory and file that refeerrnced both of those again for github pages.

Reply:
lol